### PR TITLE
IDEX-3300: Add check that workspace belongs to account owner

### PR DESCRIPTION
**Pull request by vinokurig**
Add account query parameter to getAllRunningProcesses API method.
It is needed to get all running processes that related to account.
For now if we get all running processes we will see processes that related to current user, but if there is a run process started from temporary workspace we will not see it.

### CODE REVIEW

File: platform-api/che-core-api-runner/src/test/java/org/eclipse/che/api/runner/RunnerServiceTest.java [hunk 1]
```diff
@@ -283,29 +294,78 @@ public void testGetRunningProcesses() throws Exception {
...
+        processes = service.getRunningProcesses(null, "project", null);
         assertEquals(processes.size(), 0);
 
         // no project with that workspace + name
-        processes = service.getRunningProcesses("dummy", "project");
+        processes = service.getRunningProcesses("dummy", "project", null);
         assertEquals(processes.size(), 0);
     }
 
+    /**
```
Comment (sleshchenko) [inline platform-api/che-core-api-runner/src/test/java/org/eclipse/che/api/runner/RunnerServiceTest.java]: I think javadocs is redundant in tests. Except private method and specific cases

File: platform-api/che-core-api-runner/src/main/java/org/eclipse/che/api/runner/RunnerService.java [hunk 2]
```diff
@@ -149,55 +167,82 @@ public ApplicationProcessDescriptor getStatus(@ApiParam(value = "Workspace ID",
...
         }
         return processes;
     }
 
+    private boolean isRelatedToAccountOrUser(RunQueueTask task, String accountId, String userId) throws ServerException {
+        if (accountId != null) {
+            try {
+                return accountId.equals(workspaceDao.getById(task.getRequest().getWorkspace()).getAccountId());
+            } catch (NotFoundException e) {
+                //Workspace does not exist
```
Comment (sleshchenko) [inline platform-api/che-core-api-runner/src/main/java/org/eclipse/che/api/runner/RunnerService.java]: 'Workspace does not exist' is not informative. You should write something about 'How we respond on this situation'

**Comment by sleshchenko** [comment]
Other OK for me. @skabashnyuk WDYT?

**Comment by skabashnyuk** [comment]
LGTM

**Pull request merged**